var/spack/repos/builtin/packages/tkrzw/package.py

```diff
@@ -37,4 +59,22 @@ class Tkrzw(AutotoolsPackage):
     version('0.9.2',  sha256='9040af148ab3f35c6f1d4c83f2eba8b68625dbd760f2c0537a9981dbc9bbc661')
     version('0.9.1',  sha256='1062502f93d4a9b387372d89265a9ede1704c6bcadd9aac23f5fc8383e26045a')
 
+    variant('zlib', default=False,
+            description='Enable zlib support')
+    variant('lz4', default=False,
+            description='Enable lz4 support')
+    variant('lzma', default=False,
+            description='Enable lzma support')
+    variant('zstd', default=False,
+            description='Enable zstd support')
```

Comment:
Would it make sense to have a single `compression` multi-valued variant?

Reply:
I thought about it but I wasn't sure a multivalued variant could have no value. I don't know what's best practice.

That said I just notice that I forgot an important thing: adding the corresponding `--enable-<foo>` flags to configure arguments. I'll add that shortly.

Reply:
@mdorier From `eckit`:
```python
    variant('compression',
            values=any_combination_of('bzip2', 'snappy', 'lz4', 'aec'),
            description='List of supported compression backends')
```
This allows `compression=none` to be used to express absence of a value.

Reply:
Ok, and how do I check for a particular value? i.e. what's the equivalent of `if "+zlib" in spec`?

Reply:
The equivalent would be `compression=zlib` and that matches if `zlib` is present regardless of whatever else might be present too.

Reply:
Thanks! I have changed the package accordingly.